torch/_inductor/graph.py

```diff
@@ -211,11 +218,25 @@ def __init__(
         self.sizevars = SizeVarAllocator(shape_env)
         self.graph_inputs: Dict[str, TensorBox] = {}
         self.graph_inputs_original: Dict[str, InputBuffer] = {}
-        self.device_types: Set[str] = set()
-        self.device_idxs: Set[int] = set()
+        self.device_types: Set[str] = (
+            const_module.device_types if const_module else set()
+        )
+        self.device_idxs: Set[int] = const_module.device_idxs if const_module else set()
         self.cuda = False
         self.buffers: List[ir.Buffer] = []
-        self.constants: Dict[str, torch.Tensor] = {}
+        self.const_output_index: Dict[str, int] = (
```

Comment:
Can't we infer this from the graph ? We know from the graph which output name corresponds to which name.

Reply:
I would say yes now. We can definitely match the output name that we have prefix of `_FOLDED_CONST_` and match with the node name. I've chose to err on the safe side to just pass information from upstream which constant graph is generated, but I'm also fine to hardcode the string if you prefer.